application.go

```diff
@@ -1,9 +1,18 @@
 package main
 
 import (
+	"os"
 	"project-symi-backend/app/infrastructure"
+
+	"github.com/joho/godotenv"
 )
 
 func main() {
-	infrastructure.Router.Run()
+	err := godotenv.Load()
+	print(err)
+	port := ":" + os.Getenv("PORT")
```

Comment:
In my opinion, the solution below is better. Because I don't want to "+" repeatedly.
```
port := os.Getenv("PORT")
    if len(port) == 0 {
        port = "8080"
    }
    r.Run(":" + port)
```

Reply:
changed to this format, you are right, more concise!